.circleci/config.yml

```diff
@@ -7,15 +7,7 @@ jobs:
     steps:
       - checkout
       # Node
-      - restore_cache:
-          keys:
-            - node-v1-{{ checksum "package.json" }}
-            - node-v1-
       - run: npm install
-      - save_cache:
-          key: node-v1-{{ checksum "package.json" }}
-          paths:
-            - node_modules
```

Comment:
That change is something we probably should discuss. It seems like some dependency was updated (couldn't quickly find out which one), which breaks the tests now, because the ordering of attributes in snapshots change. However, this error didn't happen on circle, only on my locale machine. When we remove that cache it also happens on circle.

The difference for the install seems to be 25 seconds (can be compared [here](https://circleci.com/gh/sulu/sulu/870) and [here](https://circleci.com/gh/sulu/sulu/871))